app/controllers/concerns/finder_top_result_ab_testable.rb

```diff
@@ -27,7 +27,7 @@ def set_test_response_header
   end
 
   def test_in_scope?
-    content_item.is_finder? && finder.eu_exit_finder?
+    content_item.is_finder? && finder_api && finder.eu_exit_finder?
```

Comment:
Why is it necessary to check for `finder_api`?

Reply:
Here's a comment I wrote and didn't commit, should I add this?

```
    # The finder is requested lazily, if the format is invalid then
    # 'finder_api' will be nil.  Rather than unconditionally request
    # the finder regardless of format (only to then serve an error
    # page), check here if the finder has been initialised before
    # trying to use it.
```

Reply:
Does `finder` instantiate `finder_api`?

Reply:
`finder_api` is instantiated in `show` to the normal finder (in the html and json branches) or the special feedy finder (in the atom branch), but it's not instantiated at all in the `ActionController::UnknownFormat` branch.

There's a test which checks an error is served if you request the finder at an invalid format, which is failing because this AB test tries to use `finder_api`, which is no longer being set.  It seems silly to request the content item from content-store to then immediately discard it and serve an error.